Sources/AppCoinsSDK/Models/Services/AptoideIOS/AptoideIOSServiceClient.swift

```diff
@@ -28,15 +28,17 @@ internal class AptoideIOSServiceClient : AptoideIOSService {
             let task = URLSession.shared.dataTask(with: request) { data, response, error in
                 if let error = error {
                     if let nsError = error as NSError?, nsError.code == NSURLErrorNotConnectedToInternet {
-                        result(.failure(.noInternet))
+                        result(.failure(.noInternet(message: "Internet Connection Failed", description: "Could not get internet connection to \(url)", request: DebugRequestInfo(request: request, responseData: data, response: response))))
```

Comment:
Also add code location on the description or service layer errors.

Reply:
Done